src/modules/common/components/nav-panel/nav-panel.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import classNames from 'classnames';
+import Styles from 'modules/common/components/nav-panel/nav-panel.styles';
+
+
+const NavPanel = p => (
+  <div className={Styles.NavPanel}>
+    <aside className={Styles.NavPanel__controls}>
+      {p.items && p.items.map((item, ind) => {
+        const Icon = item.iconComponent;
+        return (
+          <button
+            className={classNames(
+              {
+                [Styles.NavPanel__control]: true,
```

Comment:
Instead of setting this here, you could do something to the effect of: 

`classNames(`${Styles.NavPanel__control}`, {…})`